requirements.txt

```diff
@@ -272,3 +272,10 @@ argparse==1.3.0 \
     --hash=sha256:b3a79a23d37b5a02faa550b92cbbbebeb4aa1d77e649c3eb39c19abf5262da04
 diff-match-patch==20121119 \
     --hash=sha256:9dba5611fbf27893347349fd51cc1911cb403682a7163373adacc565d11e2e4c
+
+# ----------------------------------------------------------------------------------
+
+coverage==4.4.2	\
+    --hash=sha256:eea9135432428d3ca7ee9be86af27cb8e56243f73764a9b6c3e0bda1394916be
+codecov==2.0.9 \
+    --hash=sha256:ad82f054837b02081f86ed1eb6c04cddc029fbc734eaf92ff73da1db3a79188b
```

Comment:
I assume those have no sub-requirements, can you add them to the first section of this file, in alphabetical order?